text/0041-extensible-block-body.md

```diff
@@ -53,10 +53,10 @@ This can be illustrated by the following diagram:
                          /                  \
                    hash(transfer_hashes)     hash_3_______
                                             /             \
-                                         hash(proposer)   SENTINEL
+                                         hash(proposer)   SENTINEL1
 ```
 
-`SENTINEL` is a value that is very improbable to be a valid hash of some data - like, for example, the all-zeros hash (`000000...00`).
+`SENTINEL1` is a value that is very improbable to be a valid hash of some data - like, for example, the all-zeros hash (`111111...11`).
```

Comment:
Sorry, some more nitpicks :grimacing: It still says "all-zeros", which is no longer true, and `[1; 32]` isn't actually `1111...11` in hex, it's `010101...01`.